website/content/docs/drivers/docker.mdx

```diff
@@ -868,10 +868,18 @@ plugin "docker" {
   capabilities and exclusively use host based log aggregation, you may consider
   this option to disable nomad log collection overhead.
 
-- `extra_labels` - Extra labels to add to Docker containers. 
+- `extra_labels` - Extra labels to add to Docker containers.
    Available options are `job_name`, `job_id`, `task_group_name`, `task_name`,
    `namespace`, `node_name`, `node_id`. Globs are supported (e.g. `task*`)
 
+- `logging` stanza:
+
+  - `type` - Defaults to `"json-file"`. Specifies the logging driver docker should
+    use for all containers Nomad starts.
+
+  - `config` - Defaults to `{ max-file = "2", max-size = "2m" }`. This option can be
+    used to pass further configuration to the logging driver.
```

Comment:
```suggestion
- `logging` stanza:

  - `type` - Defaults to `"json-file"`. Specifies the logging driver docker
    should use for all containers Nomad starts. Note that for older versions
    of Docker, only `json-file` file or `journald` will allow Nomad to read
    the driver's logs via the Docker API, and this will prevent commands such
    as `nomad alloc logs` from functioning.

  - `config` - Defaults to `{ max-file = "2", max-size = "2m" }`. This option
    can also be used to pass further
    [configuration](https://docs.docker.com/config/containers/logging/configure/)
    to the logging driver.
```